src/lib/server/server.ts

```diff
@@ -0,0 +1,100 @@
+import express from "express";
+import cors from "cors";
+import bodyParser from "body-parser";
+import * as dotEnv from "dotenv";
+import { Route } from "./server.constants";
+import { pingHandlerFactory } from "../operations/ping/ping.handler";
+import { createUserHandlerFactory } from "../operations/create-user/create-user.handler";
+import { decryptHandlerFactory } from "../operations/decrypt/decrypt.handler";
+import { encryptHandlerFactory } from "../operations/encrypt/encrypt.handler";
+import { signHandlerFactory } from "../operations/sign/sign.handler";
+import { createBundleAndSignHandlerFactory } from "../operations/create-bundle-and-sign/create-bundle-and-sign.handler";
+import { Config } from "./config/config.utils";
+import { jwtValidatorFactory } from "../middleware/jwt-validator/jwt-validator.middleware";
+import { jwtUnusedFactory } from "../middleware/jwt-unused/jwt-unused.middleware";
+
+// TODO: Not needed in Node.js 20:
+dotEnv.config();
+
+export class OthentApp {
+  app: express.Application = express();
+
+  config: Config;
+
+  constructor(config: Config) {
+    this.config = config;
+  }
+
+  getExpressApp() {
+    return this.app;
+  }
+
+  addRouteHandlers() {
+    const { app, config } = this;
+
+    // TODO: Take a look at Multer's options:
+    // const upload = multer({});
+
+    const jwtValidator = jwtValidatorFactory() as unknown as express.Handler;
+    const jwtUnused = jwtUnusedFactory() as unknown as express.Handler;
+
+    app.use(cors({ origin: "*" }));
+
+    app.use(bodyParser.json({ limit: config.UPLOAD_LIMIT }));
+    app.use(
+      bodyParser.urlencoded({ limit: config.UPLOAD_LIMIT, extended: true }),
+    );
+
+    app.get(Route.HOME, jwtValidator, jwtUnused, pingHandlerFactory());
+
+    app.get(
+      Route.CREATE_USER,
+      jwtValidator,
+      jwtUnused,
+      createUserHandlerFactory() as unknown as express.Handler,
+    );
+
+    // TODO: Data from multer doesn't seem to be used at all, as that's coming from the JWT token:
```

Comment:
Multer wasn't actually used as the files are sent inside the ID token, so until I make the change to send them separately and only include a hash within the token, I'm removing Multer's middleware.